waku/v2/utils/namespacing.nim

```diff
@@ -31,7 +31,7 @@ proc fromString*(T: type NamespacedTopic, topic: string): NamespacingResult[Name
     # Ensures that topic starts with a "/"
     return err("invalid topic format")
   
-  ok(NamespacedTopic(application: parts[1],
+  return ok(NamespacedTopic(application: parts[1],
```

Comment:
Pull out in Temoporary variable and return `ok` of that

Reply:
done!